spec/spec_helper.rb

```diff
@@ -20,13 +19,11 @@ def fetch(key, options = {}, &blk)
     self[key] = yield
   end
 
-  def read(key)
+  def read(key, _options = {})
     self[key]
   end
 
-  def write(key, value)
+  def write(key, value, _options = nil)
```

Comment:
Updated these to match signatures of [`Activesupport::Cache::Store`](https://api.rubyonrails.org/classes/ActiveSupport/Cache/Store.html)